src/components/grid-aware/StatsBlock/StatsBlock.tsx

```diff
@@ -19,17 +19,69 @@ const StatCard = ({ number, secondaryNumber, statement }: StatCardProps) => (
   </div>
 );
 
+type AxisProp = {
+  x: string | number;
+  y: number;
+};
+
+type BarStatProps = {
+  axes: AxisProp[];
+  legend: { title: string; subtitle: string };
+  position: string;
+};
+
+const BarStatCard = ({ axes, legend, position }: BarStatProps) => {
+  const isVertical = position === "vertical";
+  const statNumbers = axes.map((axis) => axis.y);
+  const maxNumber = Math.max(...statNumbers);
+  const fillPixels = statNumbers.map(
+    (number) => `${Math.round((number / maxNumber) * 100)}px`
+  );
+
+  return (
+    <div className={s.card}>
+      <div className={`${s.barStat} ${s[position]}`}>
+        {axes.map(({ x, y }, i) => {
+          const isYMaxNumber = y === maxNumber;
+          const yAxis = y.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",");
+
+          return (
+            <div key={`${x}-${y}`} className={`${s.barItem} ${s[position]}`}>
+              <div className={`${s.xAxis} ${s[position]}`}>{x}</div>
+              <div
+                style={{
+                  width: isVertical ? "30px" : fillPixels[i],
+                  height: isVertical ? fillPixels[i] : "30px",
+                  backgroundColor: isYMaxNumber
+                    ? "var(--color-sheltertech-blue)"
+                    : "var(--color-gray-600)",
+                  opacity: isYMaxNumber ? "" : "0.5",
+                }}
+              />
+              <div className={`${s.yAxis} ${s[position]}`}>{yAxis}</div>
+            </div>
+          );
+        })}
+      </div>
+      <div className={`${s.statement} ${s[position]}`}>{legend.title}</div>
+      <div className={s.secondaryNumber}>{legend.subtitle}</div>
+    </div>
+  );
+};
+
 type StatsBlockProps = {
   title: string;
-  subtitle?: string;
-  statCards: StatCardProps[];
+  subtitle?: string | React.ReactNode;
```

Comment:
You may want to revert this if you decide to split out the bar stats, but I just wanted to point out that `ReactNode` already includes `string`, since `ReactNode` is basically meant to represent anything that can appear in a JSX (HTML) region, and `string`s can appear in JSX. This means that you could just replace this with `React.ReactNode` and it'll have the exact same type.